src-tauri/src/browser.rs

```diff
@@ -119,3 +137,38 @@ impl Browser {
         Ok(file_path)
     }
 }
+
+impl Drop for Browser {
+    fn drop(&mut self) {
+        let container_id = self.container_id.clone();
+        tokio::spawn(async move {
+            let docker_client = ContainerManager::get().await.expect("Must be initialized");
+            if let Err(e) = docker_client.kill_container(&container_id).await {
+                tracing::error!("Can't kill container {container_id}: {e}");
```

Comment:
To be consistent across the codebase

```suggestion
                error!("Can't kill container {container_id}: {e}");
```